lib/hex/netrc.ex

```diff
@@ -0,0 +1,15 @@
+defmodule Hex.Netrc do
+  @moduledoc false
+
+  alias Hex.Netrc.{Cache, Parser}
```

Comment:
```suggestion
  alias Hex.Netrc.Cache
  alias Hex.Netrc.Parser
```

please check any other build failures for older Elixir versions!

Reply:
I already started grinding my way through them. In the previous iterations of this PR (I keep force-pushing), you can see me doing away with `with` expression and `use Agent`. Fixing the latter made the compilation proceed, so I see it now complained about using this alias syntax with Elixir < 1.2. It's amazing how many quality of life improvements Elixir got in the past years. :-)

Will fix!